docs/config/environment.rst

```diff
@@ -147,11 +147,9 @@ logging of the JVM.
    The :ref:`logging configuration <conf-logging-gc>` documentation has
    the complete list of garbage collection logging environment variables.
 
-.. _basic installations: https://crate.io/docs/crate/getting-started/en/latest/install-run/basic.html
-.. _a CrateDB Linux package: https://crate.io/docs/crate/getting-started/en/latest/install-run/special/linux.html
-.. _CrateDB on Docker: https://crate.io/docs/crate/getting-started/en/latest/install-run/special/docker.html
-.. _Java options: https://docs.oracle.com/javase/7/docs/technotes/tools/windows/java.html#CBBIJCHG
+.. _basic installations: https://crate.io/docs/crate/tutorials/en/latest/install.html
+.. _CrateDB on Docker: https://crate.io/docs/crate/tutorials/en/latest/install.html#docker
 .. _environment variables: https://en.wikipedia.org/wiki/Environment_variable
-.. _expanded tarball: https://crate.io/docs/crate/getting-started/en/latest/install-run/basic.html
+.. _expanded tarball: https://crate.io/docs/crate/tutorials/en/latest/install.html#ad-hoc-installation-unix-windows
 .. _Concurrent Mark Sweep: https://docs.oracle.com/javase/10/gctuning/concurrent-mark-sweep-cms-collector.htm
 .. _G1: https://docs.oracle.com/javase/10/gctuning/garbage-first-garbage-collector.htm
```

Comment:
Should we update these links to a newer Java SE version?

Reply:
Sure, why not.

- https://docs.oracle.com/en/java/javase/11/gctuning/concurrent-mark-sweep-cms-collector.html
- https://docs.oracle.com/en/java/javase/16/gctuning/garbage-first-g1-garbage-collector1.html

> The CMS collector is deprecated. Strongly consider using the Garbage-First collector instead.

Reply:
@amotl I updated the URLs and noted that CMS is deprecated